darwin/dataset/download_manager.py

```diff
@@ -3,14 +3,14 @@
 """
```

Comment:
Many changes that appear on this diff are style changes made by black, the code formatter, not by me.  They are of note, but not relevant to the ticket in play.

Reply:
Wouldn't it be better to have separate PR with formatting, to have a clearer PR?

Reply:
Addressed now, the settings of the formatter have been changed at @tomvars suggestion, as most were just changing line length anyway.

Might still be one or two, but the main, they're just directly relevent to the code change.